File: leetcode/jewels-and-stones.cpp
//https://leetcode.com/problems/jewels-and-stones/
class Solution {
public:
    int numJewelsInStones(string J, string S) {
        int tot = 0;
        for(char c: J){
            for(char c2: S){
                if(c == c2){
                    tot++;
                }
            }
        }
        return tot;
    }
};

File: leetcode/jewels-and-stones.rs
impl Solution {
    pub fn num_jewels_in_stones(j: String, s: String) -> i32 {
        let mut tot = 0;
        for c in j.chars() {
            for c2 in s.chars() {
                if c == c2 {
                    tot = tot + 1;
                }
            }
        }
        tot
    }
}
